docs/docs/index.md

```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
 ---
 
 # Factory Orchestrator

```

Comment:
Consider adding something like this at the very very start before 'Factory Orchesators, is a cross'...:

"Built to allow device manufactures and developers to focus more on their validation and calibration software, and less on how to run, audit and manage the lifecycle of the applications..."

trying to point out 'what its for' before 'what it does'.